packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -324,6 +352,19 @@ class FluidContainer<TContainerSchema extends ContainerSchema = ContainerSchema>
 		return this.rootDataObject.initialObjects as InitialObjects<TContainerSchema>;
 	}
 
+	// TODO: should we use ReadOnlyInfo (tag: alpha legacy) or just boolean?
+	public get readOnlyInfo(): ReadOnlyInfo {
+		return this.container.readOnlyInfo;
+	}
+
+	public forceReadonly(readonly: boolean): void {
+		if (this.container.forceReadonly === undefined) {
```

Comment:
This is fine (given that it's optional), but I'd consider make it non-optional (subject to existing limitations around how we do versioning)

Reply:
I add the check because forceReadonly is optional in IContainer interface. Do you want forceReadonly to be non-optional in the interface and the class? Or you want it in a different way. A little bit confused here.